src/sections/Company/Brand/brandPage.style.js

```diff
@@ -176,9 +176,16 @@ const BrandPageWrapper = styled.section`
 	}
 
 	@media screen and (max-width: 575px) {
-		.ImgDiv, .color-code-wrapper {
+		.color-code-wrapper{
 			margin-left: 0px;
 		}
+		.ImgDiv img{
```

Comment:
how you are ensuring using this css that the image will set its dimensions accordingly based on screen size

Reply:
@hargunkaur286 ??

Reply:
Can you please review now? @sudhanshutech @iArchitSharma I have set the width to be auto so that it gets adjusted to different screen sizes and rather set the height.